presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -0,0 +1,473 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.hive;
+
+import com.google.common.primitives.Longs;
+import io.prestodb.tempto.ProductTest;
+import io.prestodb.tempto.Requires;
+import io.prestodb.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
+import io.prestodb.tempto.query.QueryExecutor;
+import io.prestodb.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalLong;
+
+import static com.facebook.presto.tests.TestGroups.HIVE_CONNECTOR;
+import static com.facebook.presto.tests.TestGroups.HIVE_TABLE_STATISTICS;
+import static com.facebook.presto.tests.utils.QueryExecutors.onHive;
+import static com.facebook.presto.tests.utils.QueryExecutors.onPresto;
+import static com.google.common.base.Verify.verify;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Requires(ImmutableNationTable.class)
+public class TestHiveBasicTableStatistics
+        extends ProductTest
+{
+    @Test(groups = {HIVE_CONNECTOR, HIVE_TABLE_STATISTICS})
+    public void testCreateUnpartitioned()
+    {
+        String tableName = "test_basic_statistics_unpartitioned_ctas_presto";
+
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        onPresto().executeQuery(format("CREATE TABLE %s AS SELECT * FROM nation", tableName));
+
+        try {
+            BasicStatistics statistics = getBasicStatisticsForTable(onHive(), tableName);
+            assertThatStatisticsArePresent(statistics);
+            assertThatStatisticsAreNonZero(statistics);
```

Comment:
`assertThatStatisticsAreNonZero` should include `assertThatStatisticsArePresent` (and then you don't need to call it explicitly)